src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterfaceSecurity.cs

```diff
@@ -163,42 +158,13 @@ private IntPtr GetMainWindowHandle()
             }
 
             return mainWindowHandle;
-#endif
         }
 
         // Determines whether we should prompt using the Console prompting
         // APIs
         private bool PromptUsingConsole()
```

Comment:
We can remove this method and simply the conditional code in the two places it is called.

Reply:
Will change